codeship-services.yml

```diff
@@ -103,3 +102,4 @@ services:
       - PROFILE_URL_FOR_TESTS=http://pwmanager/module.php/core/authenticate.php?as=profilereview-idp-no-port
       - THEME_USE=default
     working_dir: /data
+    encrypted_env_file: codeship.env.encrypted
```

Comment:
Curious that this wasn't needed before

Reply:
It's because I deleted the composer.lock file. Without the composer.lock, it tries to figure out the dependencies and ends up prompting about an unrecognized RSA signature for github.com. Providing this gives the composer process a GitHub credential which (for some reason) avoids that RSA prompt in this automated process.